packages/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -54,6 +54,8 @@ class BaseDescopeWc extends HTMLElement {
       'preview',
       'redirect-url',
       'auto-focus',
+      'form',
+      'client',
```

Comment:
I think that because we have a few async actions before the flow starts running, using props should be ok
But this is a good point, we can check it pretty easily if you like

Reply:
it is hard to deterministically say that it would be ok. it does depend on the hosting application render cycles and the network speed. leaving it to js is bound for disappointment